### PR TITLE
Create conda.md

### PR DESCRIPTION
**Pull request by WouterVanAssche**

### CODE REVIEW

**Comment by WouterVanAssche** [comment]
merge conda.md with main

File: examples/conda.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Conda Repositories
```
Comment (MaximeVdB) [inline examples/conda.md:2]: Repositories->repositories

File: examples/conda.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+parent: Examples
+nav_order: 3
+---
+
+With Artifactory, you can deploy your conda packages to a conda repository for sharing software packages within your team.
```
Comment (MaximeVdB) [inline examples/conda.md]: conda->Conda

File: examples/conda.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+To deploy your favourite software package using the artifactory UI, visit the online platform and navigate to your `CONDA-REPOSITORY`.
```
Comment (MaximeVdB) [inline examples/conda.md]: artifactory->Artifactory

File: examples/conda.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Click "deploy" to deploy your `conda-package.tar.bz2` in this conda repository.
+
+Alternatively you can use `curl -u <USERNAME>:<API-KEY> -X PUT  https://ARTIFACTORY-URL/artifactory/REPOSITORY-NAME/conda-package.tar.bz2` 
+
+To install this software package, add the following lines to your .condarc file in your homedirectory:
```
Comment (MaximeVdB) [inline examples/conda.md]: homedirectory -> home directory

File: examples/conda.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+```
+channel_alias: https://<USERNAME>:<API-KEY>@ARTIFACTORY-URL/artifactory/api/conda/REPOSITORY-NAME/
+channels:
+  - https://<USERNAME>:<API-KEY>@ARTIFACTORY-URL/artifactory/api/conda/REPOSITORY-NAME/
+default_channels:
+  - https://<USERNAME>:<API-KEY>@ARTIFACTORY-URL/artifactory/api/conda/REPOSITORY-NAME/
+```
+Now you can activate your local conda environment and use `conda install <conda-package>` to install your conda package inside your local conda environment.
```
Comment (MaximeVdB) [inline examples/conda.md]: conda -> Conda
(x3)

**Comment by MaximeVdB** [comment]
Thanks Wouter -- looks good, just some typographical comments.

**Pull request merged**